client/windows/CSpellWindow.cpp

```diff
@@ -534,13 +534,13 @@ void CSpellWindow::setCurrentPage(int value)
 void CSpellWindow::turnPageLeft()
 {
 	if(settings["video"]["spellbookAnimation"].Bool() && !isBigSpellbook)
-		CCS->videoh->playSpellbookAnimation(VideoPath::builtin("PGTRNLFT.SMK"), pos.topLeft() + Point(13, 15));
+		CCS->videoh->playSpellbookAnimation(VideoPath::builtin("PGTRNLFT.SMK"), pos.topLeft() + Point(13, 14), Point(0,0));
```

Comment:
Should `Point(0, 0)` be the default argument?

Reply:
What this Point(0,0) signifies and why we need to supply it here?

Reply:
The `preferredLogicalResolution`, which is used by `getVideoAndBackgroundRects()` if an intro is playing.